modules/BookList.js

```diff
@@ -0,0 +1,63 @@
+// eslint-disable-next-line import/no-cycle
+import { Book, nav1 } from './Book.js';
+import { DateTime } from '../node_modules/luxon/build/es6/luxon.js';
+
+export default class BookList {
```

Comment:
- [ ] Please let's refactor our classes to ES6 functions, this way, we meet the project requirements and get closer to approval. 
You may consider doing something like `export const bookList = () => {
  code
}`;